ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__amd64-nightly-4.16-upgrade-from-stable-4.15.yaml

```diff
@@ -514,8 +512,8 @@ tests:
     cluster_profile: ibmcloud-qe-2
     env:
       TEST_FILTERS: ~ConnectedOnly&;~HyperShiftMGMT&;~MicroShiftOnly&
-      UPGRADE_CHECK_RUN_TAGS: not @fips and @disconnected
-      UPGRADE_PRE_RUN_TAGS: not @fips and @disconnected
+      UPGRADE_CHECK_RUN_TAGS: '@disconnected'
+      UPGRADE_PRE_RUN_TAGS: '@disconnected'
```

Comment:
Shall we also remove the console tag on https://github.com/openshift/release/blob/e8eedfc0df5c362db263ad1ef58ddf65b9fd89fc/ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__amd64-nightly-4.16-upgrade-from-stable-4.15.yaml#L145 ?

Reply:
@jianlinliu We can detect the cluster is fips or not for all clusters to determine whether we should run fips test. And detect the basecap-none clusters to determine whether we should run console/storage tests.
For the one you linked `aws-ipi-disc-priv-sno-f28`, I don't have the background why it excludes console tests. So I just leave it as it is now.

Reply:
Got it.